tests/lib/rules/no-restricted-properties.js

```diff
@@ -116,6 +131,35 @@ ruleTester.run("no-restricted-properties", rule, {
                 message: "'anotherObject.anotherDisallowedProperty' is restricted from being used.",
                 type: "MemberExpression"
             }]
+        }, {
+            code: "foo.__proto__",
+            options: [{
+                property: "__proto__",
```

Comment:
Could you add a test similar to this one but where the property is accessed using bracket notation?
```javascript
foo['__proto__']
```